src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -5312,6 +5312,30 @@ void local()
                 Captures($"\r\n{WorkspacesResources.Variables_captured_colon} this, @this, i"));
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        [WorkItem(26101, "https://github.com/dotnet/roslyn/issues/26101")]
+        public async Task QuickInfoCapturesOnLocalFunction4()
+        {
+            await TestAsync(@"
+class C
+{
+    int field;
```

Comment:
Thinking more about this, the field isn't very interesting case. I think it would make more sense to test with two locals defined in `M()`, one being used but the outer local function, and the other used by the inner local function.

Reply:
>because fields don't need captures.

it captures `this`. I included it for quick-info to not be empty.

> one being used but the outer local function, and the other used by the inner local function.

I suppose both should be listed as captures for the outer local function and only one for the inner.

(pending)

Reply:
(resolved)